### ISSUE TITLE
binary package not working: code_coverage-extras.cmake not installed

### ISSUE THREAD

**Issue by jschleicher** [mikeferguson/code_coverage#4]
Thank you for providing the CodeCoverage cmake macro as ROS package. When I clone your repository, it works as described in the Readme.

But if I just install the binary apt package I get an error message

```
CMake Error at [mypackage]/CMakeLists.txt:11 (include):
  include could not find load file:

    CodeCoverage
```

The problem seems to be that the .deb package does not contain the `-extras` file and thus the CMAKE_MODULE_PATH is not set:
```
$ roscd code_coverage
/opt/ros/kinetic/share/code_coverage$ find
.
./cmake
./cmake/code_coverageConfig-version.cmake
./cmake/Modules
./cmake/Modules/CodeCoverage.cmake
./cmake/code_coverageConfig.cmake
./package.xml
```

While in a similar package `cmake_modules` the `.in` files are available in the installed package:
```
$ roscd cmake_modules/
/opt/ros/kinetic/share/cmake_modules$ find
.
./cmake
./cmake/cmake_modules-extras.cmake
./cmake/cmake_modulesConfig-version.cmake
./cmake/Modules
[...]
```

**Comment by mikeferguson** [comment]
Hmm, not sure why that isn't working -- but will take a look at it shortly

**Comment by mikeferguson** [comment]
Ugh.. looks like the file is name wrong -- missing the .cmake part -- ok that should be an easy fix

**Comment by mikeferguson** [comment]
should be fixed with https://github.com/mikeferguson/code_coverage/commit/ad23d84fa58bdf11d61eeb914f4b0cd909f594ca -- but I need to get a power source before I fire up the VM to test install space (which I guess never got tested).

Once I verify that, I'll cut a new release.

**Comment by mikeferguson** [comment]
tested in an install space, looks like that fix works -- releasing 0.2.2
